samples/app-identity-linking-with-sso/csharp/README.md

```diff
@@ -22,10 +22,16 @@ Bot Mapping
 
 ![Bot](IdentityLinkingWithSSO/Images/BotLinkingOptionCard.png)
 
+- Click on `Connect to facebook` or `Connect to google` button you will get a sign in card. Click on sign in button and provide your facebook or google credentials.
```

Comment:
This is just explaining the steps to click on buttons.
We need to add the explanation of functionality like how the mapping or identity linking is working with different auth.